packages/calypso-products/src/plans-list.tsx

```diff
@@ -1483,6 +1484,7 @@ const getPlanPremiumDetails = (): IncompleteWPcomPlan => ( {
 		FEATURE_PREMIUM_THEMES_V2,
 		FEATURE_WORDADS,
 		FEATURE_STYLE_CUSTOMIZATION,
+		FEATURE_STATS_PAID,
```

Comment:
This is blog onboarding plans page.
<img width="1531" alt="Screenshot 2024-01-05 at 3 04 06 PM" src="https://github.com/Automattic/wp-calypso/assets/140841/82fa4e82-0a81-4dec-be80-8fdd95f46b72">